test/SqlExtension.Tests/Integration Tests/SqlOutputBindingIntegrationTests.cs

```diff
@@ -85,8 +87,23 @@ public void QueueTriggerProductsTest()
 
             Thread.Sleep(5000);
 
-            // Function should add 100 rows
-            Assert.Equal(100, this.ExecuteScalar("SELECT COUNT(1) FROM Products"));
+            // Function should add 100 rows (excluding ProductId >= 10000 which could be added by the timer trigger function)
```

Comment:
I would rather we make a separate table for each function to use - easier to debug and keep track of state

Reply:
Does SQL bindings support different tables? I see the table defined in the function attributes but if we use different tables, we would need to create them dynamically when we set up the database.

Reply:
The table name is the first param. I'd suggest making a new table and add a new binding just to use for the trigger though since we still want the samples to demonstrate both input and output bindings for the same table 

https://github.com/Azure/azure-functions-sql-extension/blob/main/samples/SqlExtensionSamples/OutputBindingSamples/AddProduct.cs#L17

Reply:
With the redesign this is not needed any more